docs/advanced-swapping/smart-swap-options.mdx

```diff
@@ -90,30 +91,64 @@ const route = await skipClient.route({
 }
 ```
 
-### Feature: EVM Swaps
+```JSON JSON (REST API)
+{ 
+  "amount_in": "1000000",
+  "source_asset_denom": "uusdc",
+  "source_asset_chain_id": "noble-1",
+  "dest_asset_denom": "utia",
+  "dest_asset_chain_id": "celestia",
+  "cumulative_affiliate_fee_bps": "0",
+  "allow_multi_tx": true,
+  "smart_swap_options": {
+    "split_routes": true
+  }
+}
+```
 
-#### What is it
+</CodeGroup>
 
-The API supports swapping on an EVM chain, followed by a CCTP/Axelar transfer over to the Cosmos ecosystem. This allows a user to onboard on to your IBC connected chain in 1 transaction from a broad range of EVM assets, including the memecoins retail loves to hold!
+### Response Changes when using Split Routes
 
-Currently, the API supports swapping on official Uniswap V3 deployments on the following chains (and their respective chain IDs):
+We've added a new `swapType` called `SmartSwapExactCoinIn` that's returned in the `routeResponse` and `msgsDirectResponse` when the provided route is a split route. This new `swapType` has fields that allow for multiple routes, across multiple swap venues.
 
-1. Ethereum (1)
-2. Polygon (137)
-3. Optimism (10)
-4. Arbitrum One (42161)
-5. Base (8453)
-6. BNB Chain (56)
-7. Avalanche (43114)
-8. Blast (81457)
-9. Celo (42220) - SOON
+```ts
+export type SmartSwapExactCoinIn = {
+  swapVenue: SwapVenue;
+  swapRoutes: SwapRoute[];
+};
 
-#### How to use it
+export type SwapRoute = {
+  swapAmountIn: string;
+  denomIn: string;
+  swapOperations: SwapOperation[];
+};
+```
+
+## Feature: EVM Swaps
+
+Smart Swap supports bidrectional EVM swaps: go from any asset on an EVM chain to any asset on a Cosmos chain and back again. With EVM swaps, users can onboard to your IBC connected chain in 1 transaction from a broad range of EVM assets, including the memecoins retail loves to hold!
+
+Currently, the API supports swapping on official Uniswap V3 deployments on the following chains:
+
+| Network       | Chain ID |
+|---------------|----------|
+| Ethereum      | 1        |
+| Polygon       | 137      |
+| Optimism      | 10       |
+| Arbitrum One  | 42161    |
+| Base          | 8453     |
+| BNB Chain     | 56       |
+| Avalanche     | 43114    |
+| Blast         | 81457    |
+| Celo          | 42220    |
 
-Just set the `evmSwaps` flag to true in the `smartSwapOptions` object. If using the deprecated `@skip-router` library, you must be on version 5.1.0 or above (we advise you migrate over to `@skip-go/client` as soon as possible).
+### Usage
+
+Set the `evmSwaps` flag to true in the `smartSwapOptions` object. If using the deprecated `@skip-router` library, you must be on v5.1.0+ (we strongly reccomend migrating to `@skip-go/client` as soon as possible).
 
 <CodeGroup>
-```Typescript TypeScript
```

Comment:
This seems to cause rendering issues on github preview, idk if the same on mintlify